### ISSUE TITLE
Animation Duration and Size Constraints

### ISSUE THREAD

**Issue by ghost** [CatxFish/motion-filter#18]
Is there a reason why in the Filter the animation duration is limited to 5 seconds, and the Size is limited to +-8192? For the effect i'd like to do i would need ~20 seconds animation duration and extreme sizes (Source slowly zooming into view over a long period of time)

If there is no issue with that, i can also make the changes and add a pull request :)

**Comment by CatxFish** [comment]
No, there's not a technical issue for the limits. But when you use a very slow setting , some animations looks very strange , so I choose 5 seconds as limit.
